tests/unit/pages/Editor.spec.js

```diff
@@ -469,10 +470,19 @@ describe("Editor.vue", () => {
     const dialogConfirmed = jest.spyOn(Editor.methods, "dialogConfirmed");
     const confirmPublish = jest.spyOn(Editor.methods, "confirmPublish");
     const publishPlio = jest.spyOn(Editor.methods, "publishPlio");
+    const throwConfetti = jest.spyOn(Editor.methods, "throwConfetti");
     const wrapper = mount(Editor, {
       data() {
+        const confetti = require("canvas-confetti");
+        // have to create it manually as jest creates a DIV instead of CANVAS on it's own
```

Comment:
isn't this supposed to be done by `jest-canvas-mock`? If not, what exactly is it doing is not clear to me.

Reply:
delete it.